apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Spanner.V1;
+using Google.Cloud.Spanner.V1.Logging;
+
+namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// </summary>
+    public sealed class ConnectionPoolOptions
+    {
+        private ConnectionPoolOptions()
+        {
+        }
+
+        /// <summary>
+        ///     The default instance of connection pool options.
+        /// </summary>
+        public static ConnectionPoolOptions Instance { get; } = new ConnectionPoolOptions();
```

Comment:
just ease of discovery.  developers can discover these options on spannerconnection instead of having to magically know about a static class.  open to ideas to make it static while keeping usability.